guide/pooling-and-selective-consumer-service/tests/polling-and-selective-consumer_test.bal

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/mb;
+import ballerina/log;
+import ballerina/http;
+import ballerina/task;
+import ballerina/test;
+import ballerina/cache;
+import ballerina/runtime;
+
+endpoint mb:SimpleQueueSender queueSender {
+    host: "localhost",
+    port: 5672,
+    queueName: "ProcessedQueue"
+};
+endpoint mb:SimpleTopicSubscriber subscriberSetAlarmOff {
+    topicPattern: "Alarm"
+};
+endpoint mb:SimpleTopicSubscriber subscriberNotifyAuthority {
+    topicPattern: "Authority"
+};
+endpoint mb:SimpleTopicSubscriber subscriberMaintenance {
+    topicPattern: "StatusAndMaintenance"
+};
+endpoint mb:SimpleTopicSubscriber subscriberResearch {
+    topicPattern: "Research"
+};
+string[4] messageTexts;
+service<mb:Consumer> AuthorityListener bind subscriberNotifyAuthority {
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        messageTexts[0] = untaint check message.getTextMessageContent();
+    }
+}
+service<mb:Consumer> AlarmListner bind subscriberSetAlarmOff {
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        messageTexts[1] = untaint check message.getTextMessageContent();
+    }
+}
+service<mb:Consumer> MaintenanceListener bind subscriberMaintenance {
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        //messageText3 = untaint check message.getTextMessageContent();
+        messageTexts[2] = untaint check message.getTextMessageContent();
+    }
+}
+service<mb:Consumer> ResearchListener bind subscriberResearch {
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        messageTexts[3] = untaint check message.getTextMessageContent();
+    }
+}
+function messagePublisher() {
+    json requestMessage1 = { "Message": "Earth Quake of 10 Richter Scale at the loaction near Panama", "AlarmStatus":
+    "ON", "DesasterRecoveryTeamStatus": "Dispatched" };
+    string msg1 = requestMessage1.toString();
+    mb:Message message1 = check queueSender.createTextMessage(msg1);
+    var p1 = message1.setPriority(1);
+    log:printInfo("P1 is:" + check message1.getPriority());
+    _ = queueSender->send(message1);
+    runtime:sleep(5000);
+    json requestMessage2 = { "AlarmStatus": "ON" };
```

Comment:
Better to add some newlines to separate code logically. Too cluttered.

Reply:
Add some newlines to separate code logically.